docs/source/blog/arc-collaboration.md

```diff
@@ -0,0 +1,90 @@
+
+There were also a few independent bursts of development in other areas; including providing sample pupillometry data and an example workflow showing how `movement` can be used to explore different aspects of the data (e.g. position, velocity, and diameter of the pupil), and how to normalise and filter the data.
+
+A copy of the original roadmap that was [shared on Zulip](https://neuroinformatics.zulipchat.com/#narrow/channel/406001-Movement/topic/Roadmap.3A.20Spatial.20Navigation/near/495022291) is provided below.
+
+![Original feature roadmap for Jan-Feb.](../_static/blog_posts/roadmap-jan-feb-2025.png)
+
+## What's Been Introduced?
+
+In addition to the topic listed below, we have also introduced the following smaller, self-contained features to `movement`:
+
+- We have new sample datasets available, including two eye-movement tracking (pupillometry) datasets. An example of how to analyse this data using `movement` was introduced in [#429](https://github.com/neuroinformatics-unit/movement/pull/429) and can be found [here](https://movement.neuroinformatics.dev/examples/mouse_eye_movements.html#sphx-glr-examples-mouse-eye-movements-py).
+- It is now possible to scale `DataArray`s expressed in pixels into other units via the {func}`movement.transforms.scale` function. Introduced in [#384](https://github.com/neuroinformatics-unit/movement/pull/384).
+
+Additionally, we have begun planning our approach to supporting annotations in time, via what we are calling [condition arrays](https://github.com/neuroinformatics-unit/movement/issues/418).
+[RoIs](#regions-of-interest-rois) provide us with a way to annotate regions in space, and the natural next step is to provide a way of supporting annotations in time too.
+Time annotations might correspond to certain stimuli in an experiment - the introduction of a reward or a change in the environment for example - and being able to refer to the corresponding time-points in an analysis by the names of these events would be very useful.
+Being able to have these events interact seamlessly with RoIs is also highly desirable.
+At present, we have started on [one such feature](https://github.com/neuroinformatics-unit/movement/pull/421/files), for determining the time-points at which (multiple) RoIs were occupied.
+It is early days currently, but you can join in the conversation for what's planned [over on GitHub](https://github.com/neuroinformatics-unit/movement/issues/418).
+
+### Plotting Made Easier
+
+The {mod}`movement.plots` submodule has been created, which provides some helpful wrapper functions for producing some of the more common plot types that come out of the analysis of `movement` datasets.
```

Comment:
The ``{mod}`movement.plots` `` reference doens't resolve because the API docs pages are generate per file, and no such file exists. I'd suggest goig got simple monospace rendering here, and cross-referencing the functions specificly later, if you wish.